IBX-2352: Document how to set trusted_proxies when using Fastly (#1539)
ezsystems/developer-documentation: docs/guide/cache/symfony_reverse_proxy.md

    Make sure that **all** traffic always comes from the trusted proxy/load balancer,
    and that there is no other way to configure it.

> When using Fastly, you need to set ```trusted_proxies``` according to the [IP ranges used by fastly](https://docs.fastly.com/en/guides/accessing-fastlys-ip-ranges).

Reviewer: ```suggestion
When using Fastly, you need to set `trusted_proxies` according to the [IP ranges used by Fastly](https://docs.fastly.com/en/guides/accessing-fastlys-ip-ranges).
```

!!! tip

>     There is no need for setting ```trusted_proxies``` when using Fastly on Platform.sh. The Platform.sh router will
>     automatically change the source IP of requests comming from Fastly, replacing the source IP with the actuall client
>     IP and remove any X-FORWARD-... header in the request before it reaches Ibexa DXP.

Reviewer: ```suggestion
    You don't have to set `trusted_proxies` when using Fastly on Platform.sh.
    The Platform.sh router automatically changes the source IP of requests coming from Fastly,
    replacing the source IP with the actual client IP and removing any `X-FORWARD-...` header in the request before it reaches Ibexa DXP.
```
